{"metadata": {"kernelspec": {"display_name": "Bash", "language": "bash", "name": "bash"}, "language_info": {"codemirror_mode": "shell", "file_extension": ".sh", "mimetype": "text/x-sh", "name": "bash"}}, "nbformat_minor": 5, "nbformat": 4, "cells": [{"cell_type": "code", "source": "# download and install PLINK2\n# see https://www.cog-genomics.org/plink/2.0/\n# WARNING: the download link may have to be updated - I couldn't find a stable link\ncd /opt/notebooks\nwget https://s3.amazonaws.com/plink2-assets/alpha3/plink2_linux_avx2_20220814.zip\nunzip -o plink2_linux_avx2_20220814.zip\n./plink2 --version", "metadata": {"trusted": true}, "execution_count": 2, "outputs": [{"name": "stdout", "text": "--2022-09-14 12:53:12--  https://s3.amazonaws.com/plink2-assets/alpha3/plink2_linux_avx2_20220814.zip\nResolving s3.amazonaws.com (s3.amazonaws.com)... 52.216.26.22\nConnecting to s3.amazonaws.com (s3.amazonaws.com)|52.216.26.22|:443... connected.\nHTTP request sent, awaiting response... 200 OK\nLength: 9134949 (8.7M) [application/zip]\nSaving to: \u2018plink2_linux_avx2_20220814.zip.1\u2019\n\nplink2_linux_avx2_2 100%[===================>]   8.71M  14.3MB/s    in 0.6s    \n\n2022-09-14 12:53:13 (14.3 MB/s) - \u2018plink2_linux_avx2_20220814.zip.1\u2019 saved [9134949/9134949]\n\nArchive:  plink2_linux_avx2_20220814.zip\n  inflating: plink2                  \nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)\n", "output_type": "stream"}]}, {"cell_type": "code", "source": "# download and install bgenix\n# see https://enkre.net/cgi-bin/code/bgen/doc/trunk/doc/wiki/bgenix.md\ncd /opt/notebooks\nwget http://code.enkre.net/bgen/tarball/release/bgen.tgz\ntar xvfz bgen.tgz > /dev/null\ncd bgen.tgz/\n./waf configure \n./waf \n./build/test/unit/test_bgen\n./build/apps/bgenix -g example/example.16bits.bgen \u2013list\ncd /opt/notebooks", "metadata": {"trusted": true}, "execution_count": null, "outputs": []}, {"cell_type": "code", "source": "# create shortcuts to genotype directories (i.e. to avoid problems with Jupyter using blanks in file names)\n\nDIR=/mnt/project/Bulk/Imputation/UKB*imputation*from*genotype/\nln -sf $DIR /opt/notebooks/impu\n\nDIR=/mnt/project/Bulk/Genotype*Results/Genotype*calls/\nln -sf $DIR /opt/notebooks/geno\n", "metadata": {"trusted": true}, "execution_count": null, "outputs": []}, {"cell_type": "code", "source": "# get the phenotype and covariate files\n\ndx download NMRpheno_Linoleic.Acid_Polyunsaturated.Fatty.Acids.txt\ndx download NMRpheno_Linoleic.Acid_Polyunsaturated.Fatty.Acids.cova.txt\n\n# make a link to the default name  \nln -sf NMRpheno_Linoleic.Acid_Polyunsaturated.Fatty.Acids.txt pheno.txt\nln -sf NMRpheno_Linoleic.Acid_Polyunsaturated.Fatty.Acids.cova.txt cova.txt\n", "metadata": {"trusted": true}, "execution_count": 3, "outputs": [{"name": "stdout", "text": "\u001b[2K[===========================================================>] Completed 12,247,394 of 12,247,394 bytes (100%) /opt/notebooks/NMRpheno_Linoleic.Acid_Polyunsaturated.Fatty.Acids.txtt\n\u001b[2K[===========================================================>] Completed 11,449,492 of 11,449,492 bytes (100%) /opt/notebooks/NMRpheno_Linoleic.Acid_Polyunsaturated.Fatty.Acids.cova.txtt\n", "output_type": "stream"}]}, {"cell_type": "code", "source": "# look at the file content\nhead -5 pheno.txt\nhead -5 cova.txt", "metadata": {"trusted": true}, "execution_count": 4, "outputs": [{"name": "stdout", "text": "FID\tIID\tnmet1\tnmet2\tnratio\tdmet1\tdmet2\tdratio\n1000068\t1000068\t-1.06446240533014\t-1.57563779976244\t0.497911448853712\t1.44180215462887\t1.93236359143486\t-0.490561436805989\n1000214\t1000214\t-1.27363638680437\t-1.15212433593209\t-0.947445908181468\t1.37573453858316\t2.03488477906259\t-0.659150240479434\n1000839\t1000839\t0.32217083828932\t0.824580409920588\t-0.830001921599838\t1.83999988567087\t2.48318375453629\t-0.643183868865423\n1001128\t1001128\t0.436464854878345\t0.152764270605868\t0.776073961495932\t1.8716071217376\t2.33522597635918\t-0.463618854621581\nFID\tIID\tsex\tage\tage2\tcholmed\tgenoPC1\tgenoPC2\tgenoPC3\tgenoPC4\tgenoPC5\tgenoPC6\tgenoPC7\tgenoPC8\tgenoPC9\tgenoPC10\n1000068\t1000068\t2\t59\t3481\t0\t-13.3259\t2.9054\t-2.37355\t0.38822\t-1.43668\t0.547364\t2.46829\t0.229228\t-0.0457369\t-1.27121\n1000214\t1000214\t2\t48\t2304\t0\t-11.5671\t2.03669\t-2.13034\t4.28344\t-7.1727\t0.415289\t-1.14447\t-1.97925\t0.895128\t3.97226\n1000839\t1000839\t2\t61\t3721\t0\t-10.8416\t2.00904\t-0.107682\t-1.1379\t-2.46351\t0.595459\t1.99684\t-0.830125\t-1.97182\t0.338768\n1001128\t1001128\t1\t56\t3136\t0\t-11.5964\t6.74721\t-1.79153\t1.72082\t-5.03195\t-1.59372\t2.33945\t-0.875247\t2.09845\t-3.36874\n", "output_type": "stream"}]}, {"cell_type": "code", "source": "# for testing extract a region from the BGEN file\n/opt/notebooks/bgen.tgz/build/apps/bgenix -g impu/ukb22828_c11_b0_v3.bgen -incl-range 11:60000000-62000000 > test.bgen\n", "metadata": {"trusted": true}, "execution_count": 5, "outputs": [{"name": "stdout", "text": "\nWelcome to bgenix\n(version: 1.1.7, revision )\n\n(C) 2009-2017 University of Oxford\n\nBuilding query                                              :  (65410/?,2.0s,33322.6/s)\nProcessing 65410 variants                                   : [******************************] (65410/65410,14.6s,4476.9/s)\nbgenix: wrote data for 65410 variants to stdout.\n\nThank you for using bgenix.\n", "output_type": "stream"}]}, {"cell_type": "code", "source": "# testing plink\n\n./plink2 --bgen test.bgen ref-first --sample impu/ukb22828_c11_b0_v3.sample \\\n         --pheno pheno.txt --pheno-name nratio \\\n         --covar cova.txt --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10 \\\n         --out nratio \\\n         --mac 100 --maf 0.01 --hwe 1e-15 --mind 0.1 --geno 0.1 \\\n         --linear hide-covar --adjust \n\n# show the first lines of the association file\nhead -5 nratio.nratio.glm.linear\n\n# show the top hits\nhead -1 nratio.nratio.glm.linear.adjusted\n", "metadata": {"trusted": true}, "execution_count": 6, "outputs": [{"name": "stdout", "text": "PLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.log.\nOptions in effect:\n  --adjust\n  --bgen test.bgen ref-first\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio\n  --pheno pheno.txt\n  --pheno-name nratio\n  --sample impu/ukb22828_c11_b0_v3.sample\n\nStart time: Wed Sep 14 12:57:19 2022\n15544 MiB RAM detected; reserving 7772 MiB for main workspace.\nUsing up to 8 compute threads.\n--bgen: 65410 variants detected, format v1.2.\n487409 samples imported from .sample file to nratio-temporary.psam .\n--bgen: nratio-temporary.pgen + nratio-temporary.pvar written.\n487409 samples (264251 females, 222957 males, 201 ambiguous; 487409 founders)\nloaded from nratio-temporary.psam.\n65410 variants loaded from nratio-temporary.pvar.\n1 quantitative phenotype loaded (98461 values).\nCalculating sample missingness rates... done.\n0 samples removed due to missing genotype data (--mind).\n13 covariates loaded from cova.txt.\n487409 samples (264251 females, 222957 males, 201 ambiguous; 487409 founders)\nremaining after main filters.\n98461 quantitative phenotype values remaining after main filters.\nCalculating allele frequencies... done.\n--geno: 327 variants removed due to missing genotype data.\n--hwe: 4671 variants removed due to Hardy-Weinberg exact test (founders only).\n56100 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n4312 variants remaining after main filters.\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 9.19682.\n--adjust values (4312 tests) written to nratio.nratio.glm.linear.adjusted .\nEnd time: Wed Sep 14 12:58:04 2022\n#CHROM\tPOS\tID\tREF\tALT\tA1\tTEST\tOBS_CT\tBETA\tSE\tT_STAT\tP\tERRCODE\n11\t60000078\trs585540\tT\tC\tT\tADD\t98461\t-0.0141503\t0.00858698\t-1.64788\t0.0993806\t.\n11\t60000300\trs668095\tT\tC\tT\tADD\t98461\t-0.0134603\t0.00859612\t-1.56585\t0.117386\t.\n11\t60000301\trs139056520\tG\tA\tA\tADD\t98461\t0.0125146\t0.0204263\t0.612672\t0.540095\t.\n11\t60000310\trs649234\tT\tA\tT\tADD\t98461\t-0.0139806\t0.00858738\t-1.62804\t0.103519\t.\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n", "output_type": "stream"}]}, {"cell_type": "code", "source": "# run associations by chromosome\nCFROM=11\nCTO=11\n\nCFROM=1\nCTO=22\n\nexport CHR=$CFROM\nwhile [ $CHR -le $CTO ] ; do\n\n  echo \"working on CHR $CHR\"\n  \n  # run PLINK\n  \n  # using imputed data (too slow, need to reduce samples to NMR set first, and still ...)\n  # ./plink2 --bgen impu/ukb22828_c${CHR}_b0_v3.bgen ref-first --sample impu/ukb22828_c${CHR}_b0_v3.sample \\\n  \n  # use genotyped data\n  ./plink2 --bfile geno/ukb22418_c${CHR}_b0_v2 \\\n         --pheno pheno.txt --pheno-name nratio,nmet1,nmet2 \\\n         --covar cova.txt --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10 \\\n         --out nratio.${CHR} \\\n         --mac 100 --maf 0.01 --hwe 1e-15 --mind 0.1 --geno 0.1 \\\n         --linear hide-covar --adjust \n         \n  head -2 nratio.${CHR}.nratio.glm.linear.adjusted   \n  head -2 nratio.${CHR}.nmet1.glm.linear.adjusted   \n  head -2 nratio.${CHR}.nmet2.glm.linear.adjusted   \n\n\n  ((CHR=CHR+1))\n  \ndone", "metadata": {"collapsed": true, "jupyter": {"outputs_hidden": true}, "trusted": true}, "execution_count": 5, "outputs": [{"name": "stdout", "text": "working on CHR 1\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.1.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c1_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.1\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 16:44:58 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c1_b0_v2.fam.\n63487 variants loaded from geno/ukb22418_c1_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 51done.\n2 samples removed due to missing genotype data (--mind).\nIDs written to nratio.1.mindrem.id .\n13 covariates loaded from cova.txt.\n488375 samples (264744 females, 223430 males, 201 ambiguous; 488375 founders)\nremaining after main filters.\nCalculating allele frequencies... 51done.\n--geno: 4089 variants removed due to missing genotype data.\n--hwe: 9759 variants removed due to Hardy-Weinberg exact test (founders only).\n9239 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n40400 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 52done.\nResults written to nratio.1.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.12883.\n--adjust values (40400 tests) written to nratio.1.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 52done.\nResults written to nratio.1.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.11524.\n--adjust values (40400 tests) written to nratio.1.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 52done.\nResults written to nratio.1.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.06741.\n--adjust values (40400 tests) written to nratio.1.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 16:46:35 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n1\trs10889338\tT\t1.70718e-17\t1.73658e-16\t6.89701e-13\t6.89701e-13\t6.89671e-13\t6.89671e-13\t3.69405e-13\t4.13135e-12\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n1\trs1168013\tC\t2.15175e-108\t1.10105e-96\t8.69307e-104\t8.69307e-104\t8.69307e-104\t8.69307e-104\t8.69307e-104\t9.72216e-103\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n1\trs1168013\tC\t5.28886e-118\t1.85414e-106\t2.1367e-113\t2.1367e-113\t2.1367e-113\t2.1367e-113\t2.1367e-113\t2.38965e-112\nworking on CHR 2\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.2.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c2_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.2\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 16:46:36 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c2_b0_v2.fam.\n61966 variants loaded from geno/ukb22418_c2_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 52done.\n6 samples removed due to missing genotype data (--mind).\nIDs written to nratio.2.mindrem.id .\n13 covariates loaded from cova.txt.\n488371 samples (264742 females, 223428 males, 201 ambiguous; 488371 founders)\nremaining after main filters.\nCalculating allele frequencies... 52done.\n--geno: 4598 variants removed due to missing genotype data.\n--hwe: 9884 variants removed due to Hardy-Weinberg exact test (founders only).\n6843 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n40641 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 53done.\nResults written to nratio.2.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.09474.\n--adjust values (40641 tests) written to nratio.2.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 53done.\nResults written to nratio.2.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.12109.\n--adjust values (40641 tests) written to nratio.2.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 53done.\nResults written to nratio.2.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.05163.\n--adjust values (40641 tests) written to nratio.2.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 16:48:19 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n2\trs4245791\tC\t3.60849e-22\t3.69305e-21\t1.46652e-17\t1.46652e-17\t0\t0\t1.46652e-17\t1.64101e-16\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n2\trs525172\tT\t4.0901e-51\t2.67374e-47\t1.66226e-46\t1.66226e-46\t1.66226e-46\t1.66226e-46\t1.66226e-46\t1.86003e-45\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n2\trs1260326\tT\t3.13919e-83\t6.91384e-75\t1.2758e-78\t1.2758e-78\t1.2758e-78\t1.2758e-78\t1.2758e-78\t1.42759e-77\nworking on CHR 3\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.3.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c3_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.3\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 16:48:19 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c3_b0_v2.fam.\n52300 variants loaded from geno/ukb22418_c3_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 62done.\n6 samples removed due to missing genotype data (--mind).\nIDs written to nratio.3.mindrem.id .\n13 covariates loaded from cova.txt.\n488371 samples (264742 females, 223428 males, 201 ambiguous; 488371 founders)\nremaining after main filters.\nCalculating allele frequencies... 62done.\n--geno: 3640 variants removed due to missing genotype data.\n--hwe: 7799 variants removed due to Hardy-Weinberg exact test (founders only).\n6090 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n34771 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 61done.\nResults written to nratio.3.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.07532.\n--adjust values (34771 tests) written to nratio.3.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 61done.\nResults written to nratio.3.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.07288.\n--adjust values (34771 tests) written to nratio.3.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 61done.\nResults written to nratio.3.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.06366.\n--adjust values (34771 tests) written to nratio.3.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 16:50:01 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n3\trs11719420\tG\t4.01668e-05\t6.84116e-05\t1\t1\t0.75258\t0.75258\t0.452472\t1\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n3\trs2290159\tC\t3.21277e-09\t1.13492e-08\t0.000111711\t0.000111711\t0.000111705\t0.000111705\t0.000111711\t0.0012326\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n3\trs2290159\tC\t8.101e-09\t2.59085e-08\t0.00028168\t0.00028168\t0.00028164\t0.00028164\t0.00028168\t0.00310799\nworking on CHR 4\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.4.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c4_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.4\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 16:50:02 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c4_b0_v2.fam.\n47443 variants loaded from geno/ukb22418_c4_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 69done.\n5 samples removed due to missing genotype data (--mind).\nIDs written to nratio.4.mindrem.id .\n13 covariates loaded from cova.txt.\n488372 samples (264743 females, 223428 males, 201 ambiguous; 488372 founders)\nremaining after main filters.\nCalculating allele frequencies... 69done.\n--geno: 3018 variants removed due to missing genotype data.\n--hwe: 7507 variants removed due to Hardy-Weinberg exact test (founders only).\n4230 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n32688 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 67done.\nResults written to nratio.4.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.08353.\n--adjust values (32688 tests) written to nratio.4.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 67done.\nResults written to nratio.4.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.10505.\n--adjust values (32688 tests) written to nratio.4.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 67done.\nResults written to nratio.4.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.09917.\n--adjust values (32688 tests) written to nratio.4.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 16:51:55 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n4\trs13118118\tC\t3.87051e-06\t1.05802e-05\t0.126519\t0.126519\t0.118843\t0.118843\t0.077395\t0.849178\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n4\trs731450\tC\t8.96404e-11\t4.70812e-10\t2.93016e-06\t2.93016e-06\t2.93016e-06\t2.93016e-06\t2.93016e-06\t3.21497e-05\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n4\trs976002\tG\t7.04084e-12\t6.90246e-11\t2.30151e-07\t2.30151e-07\t2.30151e-07\t2.30151e-07\t2.30151e-07\t2.52521e-06\nworking on CHR 5\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.5.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c5_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.5\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 16:51:56 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c5_b0_v2.fam.\n46314 variants loaded from geno/ukb22418_c5_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 70done.\n16 samples removed due to missing genotype data (--mind).\nIDs written to nratio.5.mindrem.id .\n13 covariates loaded from cova.txt.\n488361 samples (264738 females, 223422 males, 201 ambiguous; 488361 founders)\nremaining after main filters.\nCalculating allele frequencies... 70done.\n--geno: 3699 variants removed due to missing genotype data.\n--hwe: 6881 variants removed due to Hardy-Weinberg exact test (founders only).\n4359 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n31375 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 71done.\nResults written to nratio.5.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.07664.\n--adjust values (31375 tests) written to nratio.5.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 71done.\nResults written to nratio.5.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.05938.\n--adjust values (31375 tests) written to nratio.5.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 71done.\nResults written to nratio.5.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.05483.\n--adjust values (31375 tests) written to nratio.5.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 16:53:28 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n5\trs12916\tC\t2.01066e-41\t7.85066e-40\t6.30843e-37\t6.30843e-37\t6.30843e-37\t6.30843e-37\t6.30843e-37\t6.89575e-36\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n5\trs12916\tC\t1.52851e-45\t6.90654e-43\t4.79571e-41\t4.79571e-41\t4.79571e-41\t4.79571e-41\t4.79571e-41\t5.24219e-40\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n5\trs3846663\tT\t1.03303e-27\t2.98604e-26\t3.24112e-23\t3.24112e-23\t0\t0\t1.81223e-23\t1.98095e-22\nworking on CHR 6\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.6.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c6_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.6\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 16:53:28 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c6_b0_v2.fam.\n53695 variants loaded from geno/ukb22418_c6_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 61done.\n3 samples removed due to missing genotype data (--mind).\nIDs written to nratio.6.mindrem.id .\n13 covariates loaded from cova.txt.\n488374 samples (264743 females, 223430 males, 201 ambiguous; 488374 founders)\nremaining after main filters.\nCalculating allele frequencies... 61done.\n--geno: 4166 variants removed due to missing genotype data.\n--hwe: 8802 variants removed due to Hardy-Weinberg exact test (founders only).\n5474 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n35253 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 60done.\nResults written to nratio.6.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.41301.\n--adjust values (35253 tests) written to nratio.6.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 60done.\nResults written to nratio.6.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.42578.\n--adjust values (35253 tests) written to nratio.6.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 60done.\nResults written to nratio.6.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.33293.\n--adjust values (35253 tests) written to nratio.6.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 16:55:29 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n6\trs662138\tG\t3.52324e-13\t2.98985e-10\t1.24205e-08\t1.24205e-08\t1.24205e-08\t1.24205e-08\t1.24205e-08\t1.37216e-07\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n6\trs117733303\tG\t1.619e-23\t4.18813e-17\t5.70746e-19\t5.70746e-19\t0\t0\t5.11327e-19\t5.6489e-18\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n6\trs2247056\tT\t7.14339e-24\t3.24487e-17\t2.51826e-19\t2.51826e-19\t0\t0\t2.51826e-19\t2.78206e-18\nworking on CHR 7\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.7.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c7_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.7\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 16:55:30 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c7_b0_v2.fam.\n42722 variants loaded from geno/ukb22418_c7_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 76done.\n5 samples removed due to missing genotype data (--mind).\nIDs written to nratio.7.mindrem.id .\n13 covariates loaded from cova.txt.\n488372 samples (264743 females, 223428 males, 201 ambiguous; 488372 founders)\nremaining after main filters.\nCalculating allele frequencies... 76done.\n--geno: 2658 variants removed due to missing genotype data.\n--hwe: 6804 variants removed due to Hardy-Weinberg exact test (founders only).\n5109 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n28151 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 77done.\nResults written to nratio.7.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.07833.\n--adjust values (28151 tests) written to nratio.7.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 77done.\nResults written to nratio.7.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.09895.\n--adjust values (28151 tests) written to nratio.7.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 77done.\nResults written to nratio.7.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.05436.\n--adjust values (28151 tests) written to nratio.7.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 16:56:52 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n7\trs217373\tA\t6.96897e-08\t1.51168e-07\t0.00196183\t0.00196183\t0.00195991\t0.00195991\t0.00196183\t0.0212321\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n7\trs3812316\tG\t5.50358e-19\t1.01525e-17\t1.54931e-14\t1.54931e-14\t1.55431e-14\t1.55431e-14\t1.54931e-14\t1.67676e-13\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n7\trs3812316\tG\t1.29018e-30\t1.8397e-28\t3.63199e-26\t3.63199e-26\t3.63199e-26\t3.63199e-26\t2.07497e-26\t2.24565e-25\nworking on CHR 8\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.8.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c8_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.8\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 16:56:52 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c8_b0_v2.fam.\n38591 variants loaded from geno/ukb22418_c8_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 84done.\n6 samples removed due to missing genotype data (--mind).\nIDs written to nratio.8.mindrem.id .\n13 covariates loaded from cova.txt.\n488371 samples (264742 females, 223428 males, 201 ambiguous; 488371 founders)\nremaining after main filters.\nCalculating allele frequencies... 84done.\n--geno: 2205 variants removed due to missing genotype data.\n--hwe: 6030 variants removed due to Hardy-Weinberg exact test (founders only).\n3663 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n26693 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 85done.\nResults written to nratio.8.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.17233.\n--adjust values (26693 tests) written to nratio.8.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 85done.\nResults written to nratio.8.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.1594.\n--adjust values (26693 tests) written to nratio.8.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 85done.\nResults written to nratio.8.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.10552.\n--adjust values (26693 tests) written to nratio.8.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 16:58:10 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n8\trs328\tG\t1.12683e-16\t3.14523e-15\t3.00783e-12\t3.00783e-12\t3.00782e-12\t3.00782e-12\t1.30208e-12\t1.40226e-11\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n8\trs6982502\tC\t5.22777e-37\t3.04837e-32\t1.39545e-32\t1.39545e-32\t1.39545e-32\t1.39545e-32\t1.39545e-32\t1.50281e-31\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n8\trs6982502\tC\t7.29311e-63\t6.71204e-55\t1.94675e-58\t1.94675e-58\t1.94675e-58\t1.94675e-58\t1.94675e-58\t2.09653e-57\nworking on CHR 9\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.9.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c9_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.9\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 16:58:10 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c9_b0_v2.fam.\n34310 variants loaded from geno/ukb22418_c9_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 95done.\n5 samples removed due to missing genotype data (--mind).\nIDs written to nratio.9.mindrem.id .\n13 covariates loaded from cova.txt.\n488372 samples (264742 females, 223429 males, 201 ambiguous; 488372 founders)\nremaining after main filters.\nCalculating allele frequencies... 95done.\n--geno: 2429 variants removed due to missing genotype data.\n--hwe: 5422 variants removed due to Hardy-Weinberg exact test (founders only).\n4046 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n22413 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 96done.\nResults written to nratio.9.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.08542.\n--adjust values (22413 tests) written to nratio.9.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 96done.\nResults written to nratio.9.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.07559.\n--adjust values (22413 tests) written to nratio.9.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 96done.\nResults written to nratio.9.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.06673.\n--adjust values (22413 tests) written to nratio.9.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 16:59:31 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n9\trs507666\tA\t9.38166e-13\t4.76608e-12\t2.10271e-08\t2.10271e-08\t2.10271e-08\t2.10271e-08\t2.10271e-08\t2.22775e-07\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n9\trs1883025\tT\t3.39131e-26\t2.90762e-24\t7.60094e-22\t7.60094e-22\t0\t0\t7.1844e-22\t7.61161e-21\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n9\trs1883025\tT\t1.23512e-23\t4.36132e-22\t2.76828e-19\t2.76828e-19\t0\t0\t2.0549e-19\t2.17709e-18\nworking on CHR 10\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.10.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c10_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.10\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 16:59:31 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c10_b0_v2.fam.\n38308 variants loaded from geno/ukb22418_c10_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 85done.\n6 samples removed due to missing genotype data (--mind).\nIDs written to nratio.10.mindrem.id .\n13 covariates loaded from cova.txt.\n488371 samples (264742 females, 223428 males, 201 ambiguous; 488371 founders)\nremaining after main filters.\nCalculating allele frequencies... 85done.\n--geno: 2772 variants removed due to missing genotype data.\n--hwe: 5997 variants removed due to Hardy-Weinberg exact test (founders only).\n3964 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n25575 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 85done.\nResults written to nratio.10.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.1122.\n--adjust values (25575 tests) written to nratio.10.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 85done.\nResults written to nratio.10.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.12613.\n--adjust values (25575 tests) written to nratio.10.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 85done.\nResults written to nratio.10.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.0843.\n--adjust values (25575 tests) written to nratio.10.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:00:48 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n10\trs6479896\tC\t1.09278e-16\t1.64846e-15\t2.79477e-12\t2.79477e-12\t2.79476e-12\t2.79476e-12\t1.67559e-12\t1.79734e-11\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n10\trs970548\tC\t3.63964e-09\t2.21532e-08\t9.30838e-05\t9.30838e-05\t9.30795e-05\t9.30795e-05\t7.79597e-05\t0.000836243\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n10\trs17134592\tG\t1.61281e-09\t1.30971e-08\t4.12475e-05\t4.12475e-05\t4.12466e-05\t4.12466e-05\t2.60331e-05\t0.000279247\nworking on CHR 11\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.11.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c11_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.11\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:00:48 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c11_b0_v2.fam.\n40824 variants loaded from geno/ukb22418_c11_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 80done.\n5 samples removed due to missing genotype data (--mind).\nIDs written to nratio.11.mindrem.id .\n13 covariates loaded from cova.txt.\n488372 samples (264742 females, 223429 males, 201 ambiguous; 488372 founders)\nremaining after main filters.\nCalculating allele frequencies... 80done.\n--geno: 3113 variants removed due to missing genotype data.\n--hwe: 5875 variants removed due to Hardy-Weinberg exact test (founders only).\n6443 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n25393 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 78done.\nResults written to nratio.11.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.13992.\n--adjust values (25393 tests) written to nratio.11.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 78done.\nResults written to nratio.11.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.14842.\n--adjust values (25393 tests) written to nratio.11.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 78done.\nResults written to nratio.11.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.16521.\n--adjust values (25393 tests) written to nratio.11.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:02:24 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n11\trs102275\tC\t5.83133e-2123\t9.64996e-1823\t1.48075e-2118\t1.48075e-2118\t1.48075e-2118\t1.48075e-2118\t1.48075e-2118\t1.58729e-2117\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n11\trs964184\tG\t2.51583e-161\t3.29122e-142\t6.38846e-157\t6.38846e-157\t6.38846e-157\t6.38846e-157\t6.38846e-157\t6.84808e-156\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n11\trs964184\tG\t3.28204e-153\t4.20607e-134\t8.33409e-149\t8.33409e-149\t8.33409e-149\t8.33409e-149\t8.33409e-149\t8.9337e-148\nworking on CHR 12\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.12.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c12_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.12\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:02:24 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c12_b0_v2.fam.\n37302 variants loaded from geno/ukb22418_c12_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... 87done.\n2 samples removed due to missing genotype data (--mind).\nIDs written to nratio.12.mindrem.id .\n13 covariates loaded from cova.txt.\n488375 samples (264744 females, 223430 males, 201 ambiguous; 488375 founders)\nremaining after main filters.\nCalculating allele frequencies... 87done.\n--geno: 2432 variants removed due to missing genotype data.\n--hwe: 5701 variants removed due to Hardy-Weinberg exact test (founders only).\n4841 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n24328 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': 87done.\nResults written to nratio.12.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.10082.\n--adjust values (24328 tests) written to nratio.12.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': 87done.\nResults written to nratio.12.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.12624.\n--adjust values (24328 tests) written to nratio.12.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': 87done.\nResults written to nratio.12.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.08386.\n--adjust values (24328 tests) written to nratio.12.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:03:39 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n12\trs61133374\tA\t4.65763e-07\t1.2918e-06\t0.0113311\t0.0113311\t0.0112671\t0.0112671\t0.0113311\t0.120978\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n12\trs2244608\tG\t7.76014e-16\t1.59e-14\t1.88789e-11\t1.88789e-11\t1.88789e-11\t1.88789e-11\t1.52907e-11\t1.63253e-10\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n12\trs2393791\tC\t7.39447e-18\t4.98722e-16\t1.79893e-13\t1.79893e-13\t1.79856e-13\t1.79856e-13\t1.55231e-13\t1.65734e-12\nworking on CHR 13\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.13.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c13_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.13\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:03:39 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c13_b0_v2.fam.\n26806 variants loaded from geno/ukb22418_c13_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... done.\n277 samples removed due to missing genotype data (--mind).\nIDs written to nratio.13.mindrem.id .\n13 covariates loaded from cova.txt.\n488100 samples (264607 females, 223292 males, 201 ambiguous; 488100 founders)\nremaining after main filters.\nCalculating allele frequencies... done.\n--geno: 3062 variants removed due to missing genotype data.\n--hwe: 3983 variants removed due to Hardy-Weinberg exact test (founders only).\n1965 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n17796 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': done.\nResults written to nratio.13.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.03128.\n--adjust values (17796 tests) written to nratio.13.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': done.\nResults written to nratio.13.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.06477.\n--adjust values (17796 tests) written to nratio.13.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.13.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.01484.\n--adjust values (17796 tests) written to nratio.13.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:04:35 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n13\trs74968096\tG\t9.92662e-05\t0.000111628\t1\t1\t0.829092\t0.829092\t0.725259\t1\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n13\trs9577924\tG\t1.48585e-08\t2.45205e-08\t0.000264422\t0.000264422\t0.000264387\t0.000264387\t0.000264422\t0.00274046\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n13\trs9577924\tG\t1.86736e-08\t5.03331e-08\t0.000332315\t0.000332315\t0.00033226\t0.00033226\t0.000332315\t0.0034441\nworking on CHR 14\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.14.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c14_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.14\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:04:35 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c14_b0_v2.fam.\n25509 variants loaded from geno/ukb22418_c14_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... done.\n3 samples removed due to missing genotype data (--mind).\nIDs written to nratio.14.mindrem.id .\n13 covariates loaded from cova.txt.\n488374 samples (264744 females, 223429 males, 201 ambiguous; 488374 founders)\nremaining after main filters.\nCalculating allele frequencies... done.\n--geno: 1681 variants removed due to missing genotype data.\n--hwe: 4030 variants removed due to Hardy-Weinberg exact test (founders only).\n3354 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n16444 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': done.\nResults written to nratio.14.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.\n--adjust values (16444 tests) written to nratio.14.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': done.\nResults written to nratio.14.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.\n--adjust values (16444 tests) written to nratio.14.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.14.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.07591.\n--adjust values (16444 tests) written to nratio.14.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:05:31 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n14\trs17109118\tG\t1.35642e-05\t2.73453e-05\t0.223049\t0.223049\t0.199926\t0.199926\t0.119383\t1\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n14\trs17109118\tG\t6.72384e-07\t6.72384e-07\t0.0110567\t0.0110567\t0.0109958\t0.0109958\t0.0110567\t0.113718\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n14\tAffx-52353830\tTGCCCTTC\t3.77428e-05\t3.77428e-05\t0.620643\t0.620643\t0.462408\t0.462408\t0.497682\t1\nworking on CHR 15\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.15.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c15_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.15\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:05:31 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c15_b0_v2.fam.\n24467 variants loaded from geno/ukb22418_c15_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... done.\n1 sample removed due to missing genotype data (--mind).\nID written to nratio.15.mindrem.id .\n13 covariates loaded from cova.txt.\n488376 samples (264745 females, 223430 males, 201 ambiguous; 488376 founders)\nremaining after main filters.\nCalculating allele frequencies... done.\n--geno: 1544 variants removed due to missing genotype data.\n--hwe: 4244 variants removed due to Hardy-Weinberg exact test (founders only).\n3038 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n15641 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': done.\nResults written to nratio.15.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.07398.\n--adjust values (15641 tests) written to nratio.15.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': done.\nResults written to nratio.15.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.10006.\n--adjust values (15641 tests) written to nratio.15.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.15.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.05379.\n--adjust values (15641 tests) written to nratio.15.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:06:22 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n15\trs17184256\tG\t2.77728e-05\t4.45293e-05\t0.434394\t0.434394\t0.352347\t0.352347\t0.364863\t1\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n15\trs35853021\tT\t1.275e-99\t2.35423e-93\t1.99423e-95\t1.99423e-95\t1.99423e-95\t1.99423e-95\t1.99423e-95\t2.04107e-94\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n15\trs35853021\tT\t8.06609e-148\t5.21233e-135\t1.26162e-143\t1.26162e-143\t1.26162e-143\t1.26162e-143\t1.26162e-143\t1.29125e-142\nworking on CHR 16\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.16.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c16_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.16\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:06:22 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c16_b0_v2.fam.\n28960 variants loaded from geno/ukb22418_c16_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... done.\n3 samples removed due to missing genotype data (--mind).\nIDs written to nratio.16.mindrem.id .\n13 covariates loaded from cova.txt.\n488374 samples (264743 females, 223430 males, 201 ambiguous; 488374 founders)\nremaining after main filters.\nCalculating allele frequencies... done.\n--geno: 1985 variants removed due to missing genotype data.\n--hwe: 4747 variants removed due to Hardy-Weinberg exact test (founders only).\n4647 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n17581 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': done.\nResults written to nratio.16.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.08467.\n--adjust values (17581 tests) written to nratio.16.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': done.\nResults written to nratio.16.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.09235.\n--adjust values (17581 tests) written to nratio.16.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.16.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.09398.\n--adjust values (17581 tests) written to nratio.16.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:07:27 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n16\trs7200543\tG\t2.79124e-81\t6.48676e-75\t4.90728e-77\t4.90728e-77\t4.90728e-77\t4.90728e-77\t4.13628e-77\t4.2818e-76\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n16\trs1864163\tA\t5.84958e-28\t6.62612e-26\t1.02841e-23\t1.02841e-23\t1.02841e-23\t1.02841e-23\t1.02841e-23\t1.0646e-22\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n16\trs1864163\tA\t2.77357e-33\t4.56244e-31\t4.87622e-29\t4.87622e-29\t4.87622e-29\t4.87622e-29\t4.87622e-29\t5.04778e-28\nworking on CHR 17\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.17.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c17_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.17\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:07:27 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c17_b0_v2.fam.\n28835 variants loaded from geno/ukb22418_c17_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... done.\n23 samples removed due to missing genotype data (--mind).\nIDs written to nratio.17.mindrem.id .\n13 covariates loaded from cova.txt.\n488354 samples (264732 females, 223421 males, 201 ambiguous; 488354 founders)\nremaining after main filters.\nCalculating allele frequencies... done.\n--geno: 3231 variants removed due to missing genotype data.\n--hwe: 4213 variants removed due to Hardy-Weinberg exact test (founders only).\n5053 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n16338 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': done.\nResults written to nratio.17.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.21384.\n--adjust values (16338 tests) written to nratio.17.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': done.\nResults written to nratio.17.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.14966.\n--adjust values (16338 tests) written to nratio.17.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.17.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.10605.\n--adjust values (16338 tests) written to nratio.17.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:08:19 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n17\tAffx-13927245\tC\t9.95088e-14\t1.48563e-12\t1.62577e-09\t1.62577e-09\t1.62577e-09\t1.62577e-09\t1.46357e-10\t1.50433e-09\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n17\trs72631343\tG\t2.85073e-09\t6.9904e-08\t4.65753e-05\t4.65753e-05\t4.65742e-05\t4.65742e-05\t4.65753e-05\t0.000478724\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n17\trs8070463\tT\t7.48469e-10\t9.41979e-09\t1.22285e-05\t1.22285e-05\t1.22284e-05\t1.22284e-05\t1.22285e-05\t0.00012569\nworking on CHR 18\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.18.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c18_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.18\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:08:19 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c18_b0_v2.fam.\n21962 variants loaded from geno/ukb22418_c18_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... done.\n6 samples removed due to missing genotype data (--mind).\nIDs written to nratio.18.mindrem.id .\n13 covariates loaded from cova.txt.\n488371 samples (264742 females, 223428 males, 201 ambiguous; 488371 founders)\nremaining after main filters.\nCalculating allele frequencies... done.\n--geno: 1432 variants removed due to missing genotype data.\n--hwe: 3426 variants removed due to Hardy-Weinberg exact test (founders only).\n1684 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n15420 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': done.\nResults written to nratio.18.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.08032.\n--adjust values (15420 tests) written to nratio.18.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': done.\nResults written to nratio.18.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.08916.\n--adjust values (15420 tests) written to nratio.18.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.18.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.05229.\n--adjust values (15420 tests) written to nratio.18.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:09:07 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n18\trs77960347\tG\t5.6912e-06\t9.7171e-06\t0.0877584\t0.0877584\t0.0840181\t0.0840181\t0.058031\t0.593116\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n18\trs77960347\tG\t6.48347e-50\t8.51913e-47\t9.99751e-46\t9.99751e-46\t9.99751e-46\t9.99751e-46\t9.99751e-46\t1.02181e-44\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n18\trs77960347\tG\t1.75006e-57\t2.20861e-53\t2.69859e-53\t2.69859e-53\t2.69859e-53\t2.69859e-53\t2.69859e-53\t2.75814e-52\nworking on CHR 19\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.19.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c19_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.19\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:09:07 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c19_b0_v2.fam.\n26186 variants loaded from geno/ukb22418_c19_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... done.\n56 samples removed due to missing genotype data (--mind).\nIDs written to nratio.19.mindrem.id .\n13 covariates loaded from cova.txt.\n488321 samples (264716 females, 223404 males, 201 ambiguous; 488321 founders)\nremaining after main filters.\nCalculating allele frequencies... done.\n--geno: 2703 variants removed due to missing genotype data.\n--hwe: 3910 variants removed due to Hardy-Weinberg exact test (founders only).\n6194 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n13379 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': done.\nResults written to nratio.19.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.31924.\n--adjust values (13379 tests) written to nratio.19.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': done.\nResults written to nratio.19.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.28962.\n--adjust values (13379 tests) written to nratio.19.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.19.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.15465.\n--adjust values (13379 tests) written to nratio.19.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:09:54 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n19\trs445925\tA\t5.1635e-63\t3.00894e-55\t6.90825e-59\t6.90825e-59\t6.90825e-59\t6.90825e-59\t6.90825e-59\t6.96262e-58\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n19\trs4420638\tG\t8.00651e-92\t1.94938e-70\t1.07119e-87\t1.07119e-87\t1.07119e-87\t1.07119e-87\t1.07119e-87\t1.07962e-86\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n19\trs58542926\tT\t2.00891e-91\t1.05835e-71\t2.68772e-87\t2.68772e-87\t2.68772e-87\t2.68772e-87\t2.68772e-87\t2.70887e-86\nworking on CHR 20\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.20.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c20_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.20\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:09:54 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c20_b0_v2.fam.\n19959 variants loaded from geno/ukb22418_c20_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... done.\n7 samples removed due to missing genotype data (--mind).\nIDs written to nratio.20.mindrem.id .\n13 covariates loaded from cova.txt.\n488370 samples (264742 females, 223427 males, 201 ambiguous; 488370 founders)\nremaining after main filters.\nCalculating allele frequencies... done.\n--geno: 1171 variants removed due to missing genotype data.\n--hwe: 3179 variants removed due to Hardy-Weinberg exact test (founders only).\n2273 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n13336 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': done.\nResults written to nratio.20.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.10759.\n--adjust values (13336 tests) written to nratio.20.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': done.\nResults written to nratio.20.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.11094.\n--adjust values (13336 tests) written to nratio.20.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.20.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.05814.\n--adjust values (13336 tests) written to nratio.20.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:10:45 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n20\trs2207132\tA\t4.05457e-06\t7.45881e-06\t0.0540717\t0.0540717\t0.0526359\t0.0526359\t0.0540717\t0.544798\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n20\trs2207132\tA\t3.95267e-19\t2.01263e-17\t5.27128e-15\t5.27128e-15\t5.21805e-15\t5.21805e-15\t5.27128e-15\t5.31106e-14\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n20\trs2207132\tA\t1.7392e-18\t8.54385e-17\t2.3194e-14\t2.3194e-14\t2.32037e-14\t2.32037e-14\t2.3194e-14\t2.33691e-13\nworking on CHR 21\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.21.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c21_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.21\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:10:45 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c21_b0_v2.fam.\n11342 variants loaded from geno/ukb22418_c21_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... done.\n3 samples removed due to missing genotype data (--mind).\nIDs written to nratio.21.mindrem.id .\n13 covariates loaded from cova.txt.\n488374 samples (264743 females, 223430 males, 201 ambiguous; 488374 founders)\nremaining after main filters.\nCalculating allele frequencies... done.\n--geno: 727 variants removed due to missing genotype data.\n--hwe: 1818 variants removed due to Hardy-Weinberg exact test (founders only).\n1083 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n7714 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': done.\nResults written to nratio.21.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.05411.\n--adjust values (7714 tests) written to nratio.21.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': done.\nResults written to nratio.21.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.05574.\n--adjust values (7714 tests) written to nratio.21.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.21.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.06883.\n--adjust values (7714 tests) written to nratio.21.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:11:13 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n21\trs2409396\tT\t0.000259024\t0.000409957\t1\t1\t0.864444\t0.864444\t0.819295\t1\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n21\trs2183573\tA\t0.00010767\t0.000161992\t0.830563\t0.830563\t0.564215\t0.564215\t0.274904\t1\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n21\trs2183573\tA\t3.13896e-08\t7.22663e-08\t0.00024214\t0.00024214\t0.00024211\t0.00024211\t0.00024214\t0.00230712\nworking on CHR 22\nPLINK v2.00a3.6LM AVX2 Intel (14 Aug 2022)     www.cog-genomics.org/plink/2.0/\n(C) 2005-2022 Shaun Purcell, Christopher Chang   GNU General Public License v3\nLogging to nratio.22.log.\nOptions in effect:\n  --adjust\n  --bfile geno/ukb22418_c22_b0_v2\n  --covar cova.txt\n  --covar-name age,sex,cholmed,genoPC1,genoPC2,genoPC3,genoPC4,genoPC5,genoPC6,genoPC7,genoPC8,genoPC9,genoPC10\n  --geno 0.1\n  --glm hide-covar\n  --hwe 1e-15\n  --mac 100\n  --maf 0.01\n  --mind 0.1\n  --out nratio.22\n  --pheno pheno.txt\n  --pheno-name nratio,nmet1,nmet2\n\nStart time: Wed Aug 17 17:11:13 2022\n31310 MiB RAM detected; reserving 15655 MiB for main workspace.\nUsing up to 16 threads (change this with --threads).\n488377 samples (264746 females, 223430 males, 201 ambiguous; 488377 founders)\nloaded from geno/ukb22418_c22_b0_v2.fam.\n12968 variants loaded from geno/ukb22418_c22_b0_v2.bim.\n3 quantitative phenotypes loaded.\nCalculating sample missingness rates... done.\n3 samples removed due to missing genotype data (--mind).\nIDs written to nratio.22.mindrem.id .\n13 covariates loaded from cova.txt.\n488374 samples (264743 females, 223430 males, 201 ambiguous; 488374 founders)\nremaining after main filters.\nCalculating allele frequencies... done.\n--geno: 771 variants removed due to missing genotype data.\n--hwe: 2145 variants removed due to Hardy-Weinberg exact test (founders only).\n1890 variants removed due to allele frequency threshold(s)\n(--maf/--max-maf/--mac/--max-mac).\n8162 variants remaining after main filters.\n--glm linear regression on phenotype 'nmet1': done.\nResults written to nratio.22.nmet1.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.12756.\n--adjust values (8162 tests) written to nratio.22.nmet1.glm.linear.adjusted .\n--glm linear regression on phenotype 'nmet2': done.\nResults written to nratio.22.nmet2.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.16615.\n--adjust values (8162 tests) written to nratio.22.nmet2.glm.linear.adjusted .\n--glm linear regression on phenotype 'nratio': done.\nResults written to nratio.22.nratio.glm.linear .\n--adjust: Genomic inflation est. lambda (based on median chisq) = 1.1396.\n--adjust values (8162 tests) written to nratio.22.nratio.glm.linear.adjusted .\nEnd time: Wed Aug 17 17:11:40 2022\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n22\trs738409\tG\t1.94622e-08\t1.42925e-07\t0.00015885\t0.00015885\t0.000158837\t0.000158837\t0.00015885\t0.0015225\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n22\trs4821124\tC\t3.22764e-08\t1.92416e-07\t0.00026344\t0.00026344\t0.000263405\t0.000263405\t0.000141397\t0.00135522\n#CHROM\tID\tA1\tUNADJ\tGC\tBONF\tHOLM\tSIDAK_SS\tSIDAK_SD\tFDR_BH\tFDR_BY\n22\trs2298428\tT\t6.56054e-09\t7.75985e-08\t5.35471e-05\t5.35471e-05\t5.35457e-05\t5.35457e-05\t2.89704e-05\t0.000277668\n", "output_type": "stream"}]}, {"cell_type": "code", "source": "# compute p-gain\n\ncat > compute_pgain.sh <<'EOF'\n\nset -\nfic=$1\necho \"compute p-gain for\" $fic\nfmet1=`basename $fic .nratio.glm.linear`.nmet1.glm.linear\nfmet2=`basename $fic .nratio.glm.linear`.nmet2.glm.linear\n\necho \"checking if files are present\"\n[ -f $fic ] || exit 1\n[ -f $fmet1 ] || exit 1\n[ -f $fmet2 ] || exit 1\n\necho \"checking file sizes\"\n[ `wc -l $fic | sed 's/ .*//'` -eq `wc -l $fmet1 | sed 's/ .*//'` ] || exit 1\n[ `wc -l $fic | sed 's/ .*//'` -eq `wc -l $fmet2 | sed 's/ .*//'` ] || exit 1\n\n\necho \"getting pvalues\"\ncut -f12 $fic > prat.$$\ncut -f12 $fmet1 > pmet1.$$\ncut -f12 $fmet2 > pmet2.$$\npaste pmet1.$$ pmet2.$$ prat.$$ > p3.$$\n\necho \"computing pgain\"\ncat p3.$$ | awk '{\nif ( $1 == \"P\" ) { \n  print \"PGAIN\" \n} else {\n  x1 = $1/$3\n  x2 = $2/$3\n  if (x1 > x2) {print x2} else {print x1}\n}\n}' > pgain.$$\n\npaste $fic p3.$$ pgain.$$ > pgain2.$$\n\necho \"filter significant associations\"\ncat pgain2.$$ | awk '{\nif($17 > (10/5E-8) ){print}\n}' > $fic.pgain\n\n# cleanup\nrm *.$$\n\necho \"number of records is \"`wc -l $fic.pgain`\nEOF\n\nchmod +x compute_pgain.sh\n\nfor f in nratio.*.nratio.glm.linear ; do\n\n  ./compute_pgain.sh $f\n\ndone\n\nhead -1 nratio.1.nratio.glm.linear.pgain >  nratio.glm.linear.pgain\ncat nratio.*.nratio.glm.linear.pgain | grep -v 'CHROM' | sort -g -k 17 -r >> nratio.glm.linear.pgain", "metadata": {"collapsed": true, "jupyter": {"outputs_hidden": true}, "trusted": true}, "execution_count": 59, "outputs": [{"name": "stdout", "text": "compute p-gain for nratio.1.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 2 nratio.1.nratio.glm.linear.pgain\ncompute p-gain for nratio.10.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 12 nratio.10.nratio.glm.linear.pgain\ncompute p-gain for nratio.11.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 86 nratio.11.nratio.glm.linear.pgain\ncompute p-gain for nratio.12.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.12.nratio.glm.linear.pgain\ncompute p-gain for nratio.13.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.13.nratio.glm.linear.pgain\ncompute p-gain for nratio.14.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.14.nratio.glm.linear.pgain\ncompute p-gain for nratio.15.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.15.nratio.glm.linear.pgain\ncompute p-gain for nratio.16.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 16 nratio.16.nratio.glm.linear.pgain\ncompute p-gain for nratio.17.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 64 nratio.17.nratio.glm.linear.pgain\ncompute p-gain for nratio.18.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.18.nratio.glm.linear.pgain\ncompute p-gain for nratio.19.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.19.nratio.glm.linear.pgain\ncompute p-gain for nratio.2.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.2.nratio.glm.linear.pgain\ncompute p-gain for nratio.20.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.20.nratio.glm.linear.pgain\ncompute p-gain for nratio.21.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.21.nratio.glm.linear.pgain\ncompute p-gain for nratio.22.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.22.nratio.glm.linear.pgain\ncompute p-gain for nratio.3.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.3.nratio.glm.linear.pgain\ncompute p-gain for nratio.4.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.4.nratio.glm.linear.pgain\ncompute p-gain for nratio.5.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.5.nratio.glm.linear.pgain\ncompute p-gain for nratio.6.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.6.nratio.glm.linear.pgain\ncompute p-gain for nratio.7.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.7.nratio.glm.linear.pgain\ncompute p-gain for nratio.8.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.8.nratio.glm.linear.pgain\ncompute p-gain for nratio.9.nratio.glm.linear\nchecking if files are present\nchecking file sizes\ngetting pvalues\ncomputing pgain\nfilter significant associations\nnumber of records is 1 nratio.9.nratio.glm.linear.pgain\n", "output_type": "stream"}]}, {"cell_type": "code", "source": "# upload results\n# tar cvfz nratio.tgz nratio.*.glm.linear \n# dx upload nratio.tgz\ndx upload nratio.glm.linear.pgain", "metadata": {"trusted": true}, "execution_count": 60, "outputs": [{"name": "stdout", "text": "[===========================================================>] Uploaded 22,167 of 22,167 bytes (100%) nratio.glm.linear.pgain\nID                    file-GFyYK00J13J6FX564vgx7Z45\nClass                 file\nProject               project-G6v874QJ13J49Z2B0gz538G2\nFolder                /\nName                  nratio.glm.linear.pgain\nState                 \u001b[33mclosing\u001b[0m\nVisibility            visible\nTypes                 -\nProperties            -\nTags                  -\nOutgoing links        -\nCreated               Wed Aug 17 18:44:16 2022\nCreated by            ksuhre\n via the job          job-GFyQ6B8J13J5QJX94jxP6FJX\nLast modified         Wed Aug 17 18:44:18 2022\nMedia type            \narchivalState         \"live\"\ncloudAccount          \"cloudaccount-dnanexus\"\n", "output_type": "stream"}]}]}